server/api/api.go

```diff
@@ -0,0 +1,129 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/mattermost/mattermost-plugin-autolink/server/autolink"
+)
+
+const PluginIDContextValue = "pluginid"
+
+type LinkStore interface {
+	GetLinks() []autolink.Autolink
```

Comment:
0/5 `Load/Store` rather than `Get/Save`

Reply:
Any reason for that? Most of the MM code uses `Get/Save` vs `Load/Store` I think.

Reply:
Only a personal preference. Get/Set are forever spoiled for me by Java, `Load/Store` IMO best implies the slow nature of these operations.